src/main/java/net/kodehawa/mantarobot/commands/MoneyCmds.java

```diff
@@ -59,6 +60,7 @@
 @Module
 public class MoneyCmds {
 
+    private static final NumberFormat PERCENT_FORMAT = NumberFormat.getPercentInstance();
```

Comment:
Why is this static?

Reply:
I used static here because the number format to be used to parse percentage strings is essentially constant and should be shared with all types of the class rather than created for every instance.

Reply:
There is ever only one instance of the class when registering the commands though

Reply:
That is true but I still use it in these cases out of principle

Reply:
NumberFormat is not thread safe, the best approach would be either creating one every time it's needed or having a thread local one

Reply:
I didn't realize multiple threads may be trying to access that NumberFormat. I have wrapped it within a ThreadLocal for now.